src/supertux/menu/editor_delete_level_menu.cpp

```diff
@@ -67,7 +80,10 @@ EditorDeleteLevelMenu::menu_action(MenuItem& item)
       {
         PHYSFS_delete(m_level_full_paths[id].c_str());
         delete_item(id + 2);
-        m_level_full_paths.erase(m_level_full_paths.begin() + id);
+        m_level_full_paths[id].clear();
+        m_level_names[id].clear();
+        clear();
```

Comment:
You could add `clear()` at the beginning of `refresh()`.

Reply:
`refresh` is also called in constructor and calling `clear` while menu is empty is redundant.